src/libraries/System.ComponentModel.Composition/src/Microsoft/Internal/ReflectionServices.cs

```diff
@@ -133,7 +137,8 @@ internal static IEnumerable<MethodInfo> GetAllMethods(this Type type)
         {
             IEnumerable<MethodInfo> declaredMethods = type.GetDeclaredMethods();
 
-            Type baseType = type.BaseType;
+            Type? baseType = type.BaseType;
+            Debug.Assert(baseType != null);
```

Comment:
Why not declare it non-nullable and bang the `type.BaseType` call given it is only used one time after this.

Reply:
Actually I just noticed it is used twice. Disregard 😄